UnityAds/src/main/java/com/mopub/mobileads/UnityRouter.java

```diff
@@ -24,23 +24,38 @@
 
     private static final UnityInterstitialCallbackRouter interstitialRouter = new UnityInterstitialCallbackRouter();
 
-    static boolean initUnityAds(Map<String, String> serverExtras, Activity launcherActivity) {
-        initGdpr(launcherActivity);
+    static boolean initUnityAds(Map<String, String> serverExtras, Context context) {
```

Comment:
what is this method for?

Reply:
@jindoujiao this is for initialization call without init callback

Reply:
still kept this method since it was used in `checkAndInitializeSdk` in `UnityRewardedVideo`

Reply:
`checkAndInitalizeSdk` should use listener too. 
Also, change `initUnityAds` return type from boolean to void.